src/index.html

```diff
@@ -16,7 +16,11 @@
       href="./styles/index.scss"
```

Comment:
The `href` attribute is linking to a `.scss` file. Ensure that this Sass file is compiled to a CSS file, and update the link to point to the compiled CSS file instead.